src/codeu/chat/server/LogQueue.java

```diff
@@ -0,0 +1,52 @@
+package codeu.chat.server;
+
+
+
+import com.sun.scenario.effect.impl.sw.sse.SSEBlend_SRC_OUTPeer;
+
+import java.io.*;
+import java.util.TimerTask;
+import java.util.ArrayDeque;
+import java.util.Timer;
+
+/**
+ * Created by Lysander on 6/23/17.
+ */
+public class LogQueue {
+    File transactionLog;
```

Comment:
Okay getting on this now, I think I'll go with the former and change the class declaration while also adding accessors